lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -360,26 +398,36 @@ export async function getUpdatedPackageFiles(
     }
   }
   if (!reuseExistingBranch) {
+    const lockFileMaintenancePackageFiles: FileAddition[] =
+      lockFileMaintenanceFiles.map((name) => ({
+        type: 'addition',
+        path: name,
+        contents: null,
+      }));
```

Comment:
why this is needed? 🤔

Reply:
It's just to have a common type with the other two update loops so I can use the same `getManagersForPackageFiles` and `getPackageFilesForManager` for all three.  I originally tried to fill in the `contents` field there as well so that I could factor the entire loop out into a common function, but I couldn't get it working with an `await` inside the `map` call.

Reply:
```suggestion
    const lockFileMaintenancePackageFiles: FileAddition[] =
      lockFileMaintenanceFiles.map((name) => ({
        path: name,
      }));
```

Reply:
The `type` and `contents` fields are required in `FileAddition`.